README 2.md

```diff
@@ -0,0 +1,108 @@
+![Coverage Status](./coverage.svg)
```

Comment:
@eelegiap It looks like you may have accidentally committed this copy of the README. Did you mean to do that?